tests/Dfe.EarlyYearsQualification.UnitTests/Mocks/MockContentfulServiceTests.cs

```diff
@@ -368,8 +368,6 @@ public async Task GetDateQuestionPage_PassWhenWasQualificationStartedId_ReturnsE
         result.QuestionHint.Should().Be("Test Question Hint");
         result.FutureDateErrorMessage.Should().Be("Future date error message");
         result.FutureDateErrorBannerLinkText.Should().Be("Future date error message banner link");
-        result.IncorrectFormatErrorMessage.Should().Be("Incorrect format error message banner link");
-        result.IncorrectFormatErrorBannerLinkText.Should().Be("Enter a month between 1 and 12 and a year between 1900 and $[actual-year]$");

```

Comment:
This unit test checks the initial loading of the page where we do not show any errors, so any error messages provided are not needed. It might hint at a little change to the validator to pass down null strings instead of unused error text?

Reply:
I guess the other error messages are of use in the e2e tests, and are tested _there_ if need be.